### DISCUSSION TITLE
particles format

### DISCUSSION THREAD

**Discussion by jianhaoc** [tbepler/topaz#108] [Q&A]
Hi, I see in the "topaz convert -s 8 -o data/EMPIAR-10025/processed/particles.txt data/EMPIAR-10025/rawdata/particles.txt" (https://github.com/tbepler/topaz/blob/master/tutorial/01_quick_start_guide.ipynb). I wonder what is the format of particles.txt? Is it a file containing particles of all micrographs? Thank you!

**Comment by tbepler** [comment]
Here's a sample particles file. It should contain particles for all of the micrographs you want to train on. There's also more information [here](https://github.com/tbepler/topaz#file-formats)

[sample_particles.txt](https://github.com/tbepler/topaz/files/6355104/sample_particles.txt)
